apps/entities/models/entity.py

```diff
@@ -91,7 +91,7 @@ class Meta:
         verbose_name_plural = 'Entities'
         ordering = ['name']
 
-    searchable_fields = ['name', 'aliases', 'description']
+    searchable_fields = ['id', 'type', 'name', 'aliases', 'description']
```

Comment:
@alashow, since `searchable_fields` is used to specify which fields should be checked in database queries, would it be better to add `id` and `type` elsewhere?  e.g., we could use `.only('id', 'type', *searchable_fields)`?